src/screens/HyperSwitch/HSwitchRemoteFilter.res

```diff
@@ -248,11 +248,7 @@ module RemoteTableFilters = {
       remoteFilters->Array.filter((item: EntityType.initialFilters<'t>) =>
         item.localFilter->Js.Option.isSome
       )
-    let remoteOptions =
-      filterData
-      ->getFilterFields
-      ->Array.filter(item => item.localFilter->Js.Option.isNone)
-      ->Array.filterWithIndex((_item, index) => index > 3)
```

Comment:
We can have this check in the above filter only. 

`item.localFilter->Js.Option.isNone && index > 3`